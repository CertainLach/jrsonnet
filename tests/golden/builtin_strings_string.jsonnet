{
	lstripChars_singleChar: std.lstripChars("aaabcdef", "a"),
	lstripChars_multipleChars: std.lstripChars("klmn", "kql"),
	lstripChars_array: std.lstripChars("forward", [1, "f", [], "o", "d", "for"]),

	rstripChars_singleChar: std.rstripChars("nice_boy", "y"),
	rstripChars_multipleChars: std.rstripChars("amoguass", "sa"),
	rstripChars_array: std.rstripChars("cool just cool", ["o", "l", 12.2323443]),

	stripChars_singleCharL: std.stripChars("feefoofaa", "f"),
	stripChars_singleCharR: std.stripChars("lolkekw", "w"),
	stripChars_singleChar: std.stripChars("joper jej", "j"),

	stripChars_multipleCharsL: std.stripChars("abcdefg", "cab"),
	stripChars_multipleCharsR: std.stripChars("still breathing", "gthin"),
	stripChars_multipleChars: std.stripChars("sus sus sus", "us"),

	stripChars_arrayL: std.stripChars("chel medvedo svin", ["c", 3204990, {"svin": {}}, "vi"]),
	stripChars_arrayR: std.stripChars("lach-vs-miri", ["r", "i", "craft", "is", "mine"]),
	stripChars_array: std.stripChars("UwU Lel Stosh", ["h", "U", "s", {}, [], null, "w", [1, 2, 3]]),
}
