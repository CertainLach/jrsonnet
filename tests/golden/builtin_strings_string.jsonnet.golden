{
    "lstripChars_array": "rward",
    "lstripChars_multipleChars": "mn",
    "lstripChars_singleChar": "bcdef",
    "rstripChars_array": "cool just c",
    "rstripChars_multipleChars": "amogu",
    "rstripChars_singleChar": "nice_bo",
    "stripChars_array": " Lel Sto",
    "stripChars_arrayL": "hel medvedo svin",
    "stripChars_arrayR": "lach-vs-m",
    "stripChars_multipleChars": " sus ",
    "stripChars_multipleCharsL": "defg",
    "stripChars_multipleCharsR": "still brea",
    "stripChars_singleChar": "oper je",
    "stripChars_singleCharL": "eefoofaa",
    "stripChars_singleCharR": "lolkek"
}